elections/process/README.md

```diff
@@ -91,6 +91,10 @@ The AC election includes a period after the candidates are defined
 but before the election, for candidates to answer questions from
 the community.
 
+The questions will are hosted in the Community repo's
```

Comment:
```suggestion
The questions will be hosted in the Community repo's
```

Reply:
Updated, thanks - I think I originally wrote `are` attempted to make it `will be` and ended up with neither 😅

Reply:
Been there, done that :D